compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/BIRPackageSymbolEnter.java

```diff
@@ -854,6 +868,13 @@ private void setParamSymbols(BInvokableSymbol invokableSymbol, DataInputStream d
         }
     }
 
+    private void defineParamAnnotSymbols(DataInputStream dataInStream, BVarSymbol varSymbol) throws IOException {
```

Comment:
Is this just for params or is this more generic? Asking since this is used in defining constants as well

Reply:
Sorry, my bad. Ignore 😅